Examples/Tests/ohm_solver_magnetic_reconnection/inputs_test_2d_ohm_solver_magnetic_reconnection_picmi.py

```diff
@@ -306,12 +312,16 @@ def check_fields(self):
         if not (step == 1 or step % self.diag_steps == 0):
             return
 
-        rho = simulation.fields.get("rho_fp", level=0)[...]
+        get_xp()
+
```

Comment:
`xp` is not used below. Is this a work-around or a leftover?

Reply:
Yes, it's a work-arounds, otherwise same shutdown error `context is destroyed !!!
SIGABRT`

Reply:
Ah, got it.